### ISSUE TITLE
List order is random

### ISSUE THREAD

**Issue by ygoe** [nextcloud/tasks#152]
### Steps to reproduce
1. Create multiple tasks lists

### Expected behaviour
Lists should always be in a consistent and predictable order.

### Actual behaviour
Every time I look, the order of the lists is different.

**Comment by mikejtl** [comment]
I have a similar problem.  I'm running Tasks on Firefox for MacOS and Ubuntu 17.10. Would also love to be able to reorder lists somehow. 

Steps to reproduce
    Create a new task list or calendar.

Expected behaviour 
  Lists should always be in a consistent and predictable order with new lists added to the bottom.

Actual behaviour
A new list rearranges the order of the previously existing lists.

**Comment by Svarto** [comment]
Same for me.

**Comment by raimund-schluessler** [comment]
What order would you prefer? We could order it by the ID of the lists, so renaming will not change the order (but adding/deleting lists might do). The other option would be to order it alphabetically by their name (this might be more transparent to the user, but renaming will change the order).

Dynamically reordering lists manually will come later.

**Comment by ygoe** [comment]
Just order by name. I'm used to have things move around when I dare to change their name. But the rest of the time, it's in the same place. And I can quickly find things that are ordered by name. Phone book order, you know? Since 1878. ;-)

**Comment by mikejtl** [comment]
I agree.  Until manual reordering is available, alphabetical would be preferred. It would be obvious and not unexpected.  Users could then create their own reordering system by adding numbers or letters as prefixes to their lists. 

Future Request: smart lists would be a killer feature that most other task management tools don't offer.

**Comment by mikejtl** [comment]
Thanks for sharing this news. 

By smart lists I meant a feature like the smart collections feature, but where the criteria is defined by the user.  So, for example, I could create a list called "email" and it would search every task from across all my lists for the word "email" and put those tasks on the "email" list regardless of what other lists those tasks are currently on.  Users could then tag their tasks with various "contexts" (that's the word used in GTD) and the app would sort tasks accordingly. I hope that made sense. I'm happy to explain further if it wasn't clear.
